packages/botonic-react/src/message-utils.js

```diff
@@ -22,10 +22,19 @@ export const MEDIA_TYPES = [
 export const isMedia = message =>
   MEDIA_TYPES.some(type => isType(message.type, type))
 
+// TODO: Centralize handling attachments in class
+
 export const toBase64 = file =>
   new Promise((resolve, reject) => {
     const reader = new FileReader()
     reader.readAsDataURL(file)
     reader.onload = () => resolve(reader.result)
     reader.onerror = error => reject(error)
   })
+
+export const isAllowedSize = fileSize => {
+  const MAX_MB_SIZE_ALLOWED = 10
```

Comment:
in my personal convention I always put the units last (MAX_SIZE_ALLOWED_MB).
pedantic: word ALLOWED is not required here if you already have MAX in the name and you're within a isAllowedSize method